vocabularies/Common.xml

```diff
@@ -822,8 +829,14 @@ The minimum structure is
         </Property>
       </ComplexType>
       <ComplexType Name="ValueListParameterInOut" BaseType="Common.ValueListParameter">
+        <Property Name="Operator" Type="UI.SelectionRangeOptionType" DefaultValue="0" Nullable="false">
```

Comment:
Define new subtype `ValueListRangeInOut` of `ValueListParameterInOut`.

Reply:
=> 2e24e35